app/assets/javascripts/admin/addon/templates/admin-badges/show.hbs

```diff
@@ -65,7 +65,7 @@
     </form.Field>
 
     <form.ConditionalContent
-      @activeName={{if data.image "upload-image" "choose-icon"}}
+      @activeName={{if data.image_url "upload-image" "choose-icon"}}
```

Comment:
Maybe I could keep the alias https://github.com/discourse/discourse/blob/62e3171484e287640e7c55592d40b57f0a46143e/app/assets/javascripts/discourse/app/models/badge.js#L72-L72 now that I added `image_url` to show.js, but I feel like it's clearer not to use an alias in this case